src/radical/pilot/configs/resource_tacc.json

```diff
@@ -1,5 +1,42 @@
 
 {
+    "frontera_interactive": {
+        "description"                 : "Petascale computing system at the Texas Advanced Computing Center (TACC)",
+        "notes"                       : "Specify this resource in the pilot description and run RP within a compute node(s)",
+        "schemas"                     : ["local", "ssh"],
+        "ssh"                         :
+        {
+            "job_manager_endpoint"    : "ssh://localhost/",
+            "filesystem_endpoint"     : "sftp://localhost/"
+        },
```

Comment:
could be removed (and a corresponding value from `schemas`), since we do local runs anyway

Reply:
fixed